### DISCUSSION TITLE
Cannot find name RadioCards with NextJS

### DISCUSSION THREAD

**Discussion by Aloysius999** [radix-ui/themes#550] [Q&A]
I have tried to test the sample radio cards at

[https://www.radix-ui.com/themes/docs/components/radio-cards]

and my NextJS installation can't find the RadioCards components in @radix-ui/themes

Any ideas on where these components are located? In which module? They don't appear to be in RadixUI Themes despite the documentation.

Trying 

```
import {
  RadioCards,
} from "@radix-ui/themes";
```

like in the RadixUI source test modeuls, gives me the error

`Module '"@radix-ui/themes"' has no exported member 'RadioCards'.`

I'm using

- Latest RadixUI
- latest version of NextJS
- latest TypeScript

Thanks

![image](https://github.com/user-attachments/assets/71acb74a-82e9-4455-ac73-f599a50c6c15)

Ed - similar problem with Radio - undefined in `@radix-ui/themes` and not exported from `@radix-ui/react-radio-group`

**Comment by vladmoroz** [comment]
Do you have `"@radix-ui/themes": "latest"` in your package.json? If yes, you are probably not using the actual latest version, but the one saved in your lockfile at the time of the original installation. I'd recommend pinning Radix Themes to the "3.1.1" version instead.

**Reply by Aloysius999** [reply]
Many thanks - that worked.

I could have sworn that I'd installed the latest `@radix-ui/themes`

**Reply by theemew** [reply]
I ran into this too on a fresh `npm install @radix-ui/themes`. Adding `"@radix-ui/themes": "latest"` to package.json and running `npm install fixed it for me`
